src/Router/RabbitMq/Adapter.php

```diff
@@ -52,58 +76,75 @@ public function __invoke(Message $message)
 
 //        $subjectObs->subscribe($this->logger);
 
-        $subjectObs
+        if (null !== $this->timeout) {
             // Give only x ms to execute
-            ->timeout($this->timeout)
+            $subjectObs->timeout($this->timeout);
```

Comment:
`$subjectObs = $subjectObs->timeout($this->timeout);`

Reply:
Alright 👍